### PR TITLE
Fix the chart of accounts generation date

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR fix the date helper for handlerbars when the date format is not given totally

### CODE REVIEW

File: server/lib/template/helpers/dates.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const TIMESTAMP_FMT = 'DD/MM/YYYY HH:mm:ss';
  * @returns {String} - the formatted string for insertion into templates
  */
 function date(value, dateFormat) {
-  dateFormat = dateFormat.name === 'date' ? DATE_FMT : dateFormat;
+  dateFormat = (!dateFormat || dateFormat.name === 'date') ? DATE_FMT : dateFormat;
```
Comment (jniles) [inline server/lib/template/helpers/dates.js:20]: This is a bit confusing to read with `!` and `||`.  I think this is a better way of expressing the same thing:

``` js
dateFormat = (dateFormat && dateFormat.name !== 'date') || DATE_FMT;
```

Then it's clear to me what is default.  What do you think?

Comment (mbayopanda) [reply server/lib/template/helpers/dates.js:20]: The `dateFormat = (dateFormat && dateFormat.name !== 'date') || DATE_FMT;` work well with the account module but broke the cashflow report.

Comment (jniles) [reply server/lib/template/helpers/dates.js:20]: Ha!  Okay.  Whoops.

**Comment by jniles** [comment]
LGTM. :+1:

**Pull request merged**